File: wel_summamove_flutter/lib/models/prestatie.dart
class Prestatie {
  final int id;
  final int oefeningId;
  final String datum;
  final String starttijd;
  final String eindtijd;
  final int aantal;

  Prestatie({
    required this.id,
    required this.oefeningId,
    required this.datum,
    required this.starttijd,
    required this.eindtijd,
    required this.aantal,
  });

  factory Prestatie.fromJson(Map<String, dynamic> json) {
    return Prestatie(
      id: json['id'],
      oefeningId: json['oefening_id'],
      datum: json['datum'],
      starttijd: json['starttijd'] ?? '',
      eindtijd: json['eindtijd'] ?? '',
      aantal: json['aantal'],
    );
  }
}

File: wel_summamove_laravel/app/Models/Prestatie.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Prestatie extends Model
{
    protected $table = 'prestaties';

    protected $fillable = ['gebruiker_id',
        'oefening_id',
        'datum',
        'starttijd',
        'eindtijd',
        'aantal',
    ];

    public function gebruiker() {
        return $this->belongsTo(User::class, 'gebruiker_id');
    }
    public function oefening() {
        return $this->belongsTo(Oefening::class, 'oefening_id');
    }
}
